tests/stub/routing/test_routing_v4x3.py

```diff
@@ -212,7 +212,26 @@ def test_should_fail_when_reading_from_unexpectedly_interrupting_reader_using_se
         self._readServer1.done()
         self.assertTrue(failed)
 
-    def test_should_fail_when_reading_from_unexpectedly_interrupting_reader_using_tx_run(self):
+    @driver_feature(types.Feature.OPT_PULL_PIPELINING)
+    def test_should_fail_when_reading_from_unexpectedly_interrupting_reader_using_session_run(
+            self):
+        self._should_fail_when_reading_from_unexpectedly_interrupting_reader_using_session_run(
+            interrupting_reader_script=
+            "reader_with_unexpected_interruption_on_pipelined_pull.script"
```

Comment:
```suggestion
            "reader_with_unexpected_interruption_on_pipelined_pull.script"
```
I prefer to not name positional args. Plus, this fixes the unsatisfyingly long line.